lib/std/special/docs/index.html

```diff
@@ -5,62 +5,228 @@
     <title>Documentation - Zig</title>
     <link rel="icon" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAgklEQVR4AWMYWuD7EllJIM4G4g4g5oIJ/odhOJ8wToOxSTXgNxDHoeiBMfA4+wGShjyYOCkG/IGqWQziEzYAoUAeiF9D5U+DxEg14DRU7jWIT5IBIOdCxf+A+CQZAAoopEB7QJwBCBwHiip8UYmRdrAlDpIMgApwQZNnNii5Dq0MBgCxxycBnwEd+wAAAABJRU5ErkJggg==">
     <style type="text/css">
-      body {
-        font-family: system-ui, -apple-system, Roboto, "Segoe UI", sans-serif;
-        max-width: 60em;
+      :root {
+        font-size: 1em;
+        --ui: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
+        --mono: "Source Code Pro", monospace;
+        --tx-color: #141414;
+        --bg-color: #ffffff;
+        --link-color: #2A6286;
+        --sidebar-sh-color: rgba(0, 0, 0, 0.09);
+        --sidebar-pkg-bg-color: #f1f1f1;
+        --sidebar-pkglnk-tx-color: #141414;
+        --sidebar-pkglnk-tx-color-hover: #fff;
+        --sidebar-pkglnk-tx-color-active: #000;
+        --sidebar-pkglnk-bg-color: transparent;
+        --sidebar-pkglnk-bg-color-hover: #555;
+        --sidebar-pkglnk-bg-color-active: #FFBB4D;
+        --search-bg-color: #f3f3f3;
+        --search-bg-color-focus: #ffffff;
+        --search-sh-color: rgba(0, 0, 0, 0.18);
+        --help-sh-color: rgba(0, 0, 0, 0.75);
       }
+
+      a {
+        text-decoration: none;
+      }
+      
+      a:hover {
+        text-decoration: underline;
+      }
+      
       .hidden {
         display: none;
       }
-      a {
-        color: #2A6286;
+
+      /* layout */
+      .canvas {
+        width: 100vw;
+        height: 100vh;
+        overflow: hidden;
+        margin: 0;
+        padding: 0;
+        font-family: var(--ui);
+        color: var(--tx-color);
+        background-color: var(--bg-color);
       }
-      pre{
-        font-family:"Source Code Pro",monospace;
-        font-size:1em;
-        background-color:#F5F5F5;
-        padding:1em;
-        overflow-x: auto;
+
+      .flex-main {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        justify-content: center;
+        
+        z-index: 100;
       }
-      code {
-        font-family:"Source Code Pro",monospace;
-        font-size:1em;
+
+      .flex-filler {
+        flex-grow: 1;
+        flex-shrink: 1;
       }
-      nav {
-        width: 10em;
-        position: fixed;
-        left: 0;
-        top: 0;
-        height: 100vh;
+
+      .flex-left {
+        width: 12rem;
```

Comment:
what is `rem`?

Reply:
"root" em. Not affected by changing target element's font-size.

Chrome 4+, Firefox 3.6+, Safari 5+.